arch/avr/include/avr/irq.h

```diff
@@ -203,6 +203,21 @@ static inline void up_irq_restore(irqstate_t flags)
  * Public Function Prototypes
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: up_getusrpc
+ ****************************************************************************/
+
+#if defined(REG_PC2)
+#  define up_getusrpc(regs) \
+    ((((uint8_t *)((regs) ? (regs) : g_current_regs))[REG_PC0] << 16) | \
```

Comment:
```suggestion
    ((((uint8_t *)((regs != NULL) ? (regs) : CURRENT_REGS))[REG_PC0] << 16) | \
```

Reply:
CURRENT_REGS doesn't exist anymore， should use up_current_reg instead.

Reply:
If so, why other architectures still use CURRENT_REGS